assemblies/resources/build.xml

```diff
@@ -34,6 +34,10 @@
     <replaceregexp file="target/assembly/etc/startup.properties" match="pax-logging-log4j2/1.11.6" replace="pax-logging-api/1.11.11" byline="true"/>
     <replaceregexp file="target/assembly/bin/instance" match="1.11.6/pax-logging-api-1.11.6" replace="1.11.11/pax-logging-api-1.11.11" byline="true"/>
     <replaceregexp file="target/assembly/bin/shell" match="1.11.6/pax-logging-api-1.11.6" replace="1.11.11/pax-logging-api-1.11.11" byline="true"/>
```

Comment:
The lines above need to get updated as well

Reply:
I'll amend 6c6ed005e0b54f9970918c757dd432228bc33041.